nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md

```diff
@@ -19,7 +19,7 @@
 <!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
 **{{ cookiecutter.name }}** is a bioinformatics best-practise analysis pipeline. {{ cookiecutter.description }}
 
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses containers from [BioContainers](https://biocontainers.pro/) making installation trivial and results highly reproducible.
```

Comment:
Do we want to suggest this given that some pipelines may require containers not on Biocontainers? My thoughts are to leave it generic for now and mention it in free text somewhere else if it is indeed the case?

Reply:
Agreed 👍🏻

Reply:
@MaxUlysse ok if we close this PR then?

Reply:
No, there is still the issue on Docker containers, we now (meaning here in DSL2) do not provide containers anymore.
(except specific case).
We mainly use available containers.
But I'm ok for a change from Docker -> Docker and/or Singularity

Reply:
I'd say that even if we're not building the containers, the pipeline still comes with containers..?

```suggestion
The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker / Singularity containers making installation trivial and results highly reproducible.
```